ParisBNF/abb/BNFabb3.xml

```diff
@@ -0,0 +1,327 @@
+
+                        </support>
+                        <extent>
+                           <measure unit="leaf">129</measure>
+                           <measure unit="leaf" type="blank">1</measure>
+                           <locus target="#105r"/>
+
+                           <dimensions type="outer" unit="mm">
+                              <height>198</height>
+                              <width>123</width>
+                           </dimensions>
+                        </extent>
+                        <foliation/>
+                        <collation>
+                        </collation>
+                        <condition key="good"/>
+                     </supportDesc>
+                     <layoutDesc>
+                        <layout columns="1" writtenLines="26">
+                           <locus from="1r" to="114v"/>
+                        </layout>
+                        <layout columns="2" writtenLines="26">
+                           <locus from="115v" to="128v"/>
+                        </layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  <handDesc>
+                     <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="script">18th-century script</seg>
+                        <date notBefore="1700" notAfter="1799"/>
+                        <seg type="ink">Black, red</seg>
+                        </handNote>
+                  </handDesc>
+
+                  <additions>
+                     <list>
+                        <item xml:id="a1">
+                           <locus target="#128r"/>
+                           <desc type="Unclear">
+                             A note related to tributes,
+                             possibly in the region of <placeName ref="LOC1572Asgade"/>.</desc>
+                          <q xml:lang="am">
+                            ከ<placeName ref="LOC1572Asgade">አስገዴ፡</placeName> ፭፻፵፱ለሸለቃ፡ ሽቴ፡ ፫፻፺፭ለአዕበዮ፡ አራ፡
+                            ፬፻፸፩ለገብራይ፡ አድአቀይት፡ ፬፻፹፫ለአሰላፊ፡ ወልደ፡ ሚካኤል፡ <sic resp="DN">፫፻፺፮ለወድ፡</sic> ሸለቃ፡
+                             ፍሥሐ፡ ፫፻፷፪ለተስፋይ፡ ግባ፡ ደግ፡ <sic resp="DN">፬፻ለወድ፡</sic> ሸለቃ፡ ፍሥሐ፡ ገብረ፡ ሚካኤል።
+                              </q>
+                              <note>French translation in <bibl><ptr target="bm:ContiRossini1914Notice"/>
+                            <citedRange unit="page">49</citedRange></bibl>; words <foreign>waddi</foreign>
+                          and <foreign>ʾasallāfi</foreign> are noted as Tǝgrǝñña forms.</note>
+                        </item>
+
+                        <item xml:id="a2">
+                           <locus target="#129r"/>
+                           <desc type="Unclear">A note concerning the <foreign xml:lang="gez">rim</foreign>-land of a
+                            a church, from the time of the king <persName ref="PRS9142TaklaGi">Takla Giyorgis</persName></desc>
+                           <q xml:lang="gez">
```

Comment:
am, not gez?

Reply:
I would say, "am", but it is an issue in itself, you are right.  The note is very brief, there are mainly Amharic words and names (tigrigna or geez), but no syntactical structures and no verbs.  the preposition shows that it is not Tigrigna. Out of Geez and Amh I opted for the latter, as it is more typical.

Reply:
From what I see here, you opted for Geez :)

Reply:
changed